js/about/preferences.js

```diff
@@ -1248,16 +1219,17 @@ class PaymentsTab extends ImmutableComponent {
               <td>
                 <SettingsList>
                   <SettingItem>
-                    <select className='form-control fundsSelectBox'
+                    <FormDropdown
```

Comment:
Was this applied here in some way? https://github.com/brave/browser-laptop/blob/baff549a8e56b86f8bc0a45f80869a2c35281c60/less/about/preferences.less#L1101